kite-morphlines/kite-morphlines-core/src/test/resources/test-morphlines/hashCommand.conf

```diff
@@ -15,10 +15,11 @@
 morphlines : [
   {
     id : morphline1
-    importCommands : ["org.kitesdk.**", "org.cloudera.tristan.morphlines.**"]
+    importCommands : ["org.kitesdk.**"]
     
     commands : [    
-    { logDebug { format : "input record: {}", args : ["@{}"] } }                 
+    { 
+      logDebug { format : "input record: {}", args : ["@{}"] } }                 
```

Comment:
Can we move the closing brace } to the same indentation level as the opening brace { ?